tests/ui/async-await/in-trait/async-generics.stderr

```diff
@@ -2,35 +2,27 @@ error[E0311]: the parameter type `U` may not live long enough
   --> $DIR/async-generics.rs:9:28
    |
 LL |     async fn foo(&self) -> &(T, U);
-   |                            ^^^^^^^
+   |                  -         ^^^^^^^ ...so that the reference type `&(T, U)` does not outlive the data it points at
+   |                  |
+   |                  the parameter type `U` must be valid for the anonymous lifetime as defined here...
    |
-note: the parameter type `U` must be valid for the anonymous lifetime as defined here...
-  --> $DIR/async-generics.rs:9:18
+help: consider adding an explicit lifetime bound...
    |
-LL |     async fn foo(&self) -> &(T, U);
-   |                  ^
-note: ...so that the reference type `&(T, U)` does not outlive the data it points at
-  --> $DIR/async-generics.rs:9:28
-   |
-LL |     async fn foo(&self) -> &(T, U);
-   |                            ^^^^^^^
+LL |     async fn foo<'a>(&'a self) -> &'a (T, U) where U: 'a;
+   |                 ++++  ++           ++        +++++++++++
 
 error[E0311]: the parameter type `T` may not live long enough
   --> $DIR/async-generics.rs:9:28
    |
 LL |     async fn foo(&self) -> &(T, U);
-   |                            ^^^^^^^
+   |                  -         ^^^^^^^ ...so that the reference type `&(T, U)` does not outlive the data it points at
+   |                  |
+   |                  the parameter type `T` must be valid for the anonymous lifetime as defined here...
    |
-note: the parameter type `T` must be valid for the anonymous lifetime as defined here...
-  --> $DIR/async-generics.rs:9:18
+help: consider adding an explicit lifetime bound...
```

Comment:
One more nit. This help no longer needs `...` since it's not connected to either segment fragment above. Can you just change it to "consider adding an explicit lifetime bound", then r=me?